driver-sync/src/test/functional/com/mongodb/client/ClientSideOperationTimeoutTest.java

```diff
@@ -86,13 +86,24 @@ public static void checkSkipCSOTTest(final String fileDescription, final String
         assumeFalse("TODO (CSOT) - JAVA-4052", fileDescription.startsWith("legacy timeouts behave correctly for retryable operations"));
 
         assumeFalse("TODO (CSOT) - JAVA-4063", testDescription.contains("RTT"));
-        assumeFalse("TODO (CSOT) - JAVA-4059", fileDescription.contains("GridFS"));
 
         assumeFalse("TODO (CSOT) - JAVA-5248",
                 fileDescription.equals("MaxTimeMSExpired server errors are transformed into a custom timeout error"));
 
         assumeFalse("TODO (CSOT) - JAVA-4062", testDescription.contains("wTimeoutMS is ignored"));
 
+        if (fileDescription.contains("GridFS")) {
```

Comment:
Whats happening with these tests?

Reply:
Unfortunately, there is a dependency on CRUD work, and the `maxTimeMs` parameter is not included in many operations, which is causing the tests to fail. I will add a //TODO [JAVA-4057](https://jira.mongodb.org/browse/JAVA-4057)